### DISCUSSION TITLE
1.27.0

### DISCUSSION THREAD

**Discussion by Koenkk** [Koenkk/zigbee2mqtt#13356] [General]
# Features
- #12988 Add `filtered_cache` option ([docs](https://www.zigbee2mqtt.io/guide/configuration/devices-groups.html#common-device-options))

# Improvements
- #13047 Allow regexes for `filtered_attributes`, `filtered_cache` and `filtered_optimistic` options
- [#4046](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4046) Expose more features for Plugwise 106-03
- #12952 Expose `power_outage_memory` and `switch_type` for Lonsonho TS110E_2gang
- #8499 Revert back to Xiaomi battery linear % (2850-3200mv) calculation
- #6181 Expose `pin_code` for Weiser 9GED21500-005
- [#4413](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4413) Expose `led_disabled_night` and `power_outage_count` for Xiaomi WS-EUK03 and WS-EUK04
- [#4414](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4414) Support `brightness_move` and `brightness_step` for Ubisys D1 
- [#4419](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4419) Expose more features for Xiaomi WS-USC01
- [#4421](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4421) Support OTA for Gledopto GL-D-006P
- [#4426](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4426) Expose `power_on_behavior` and `indicator_mode` for Tuya `TS0111_valve` and expose `power_on_behavior` for Tuya `ZN231392`
- #12230 Expose `action_user` for Yale locks
- [#4431](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4431) Expose `brightness_level` for TuYa S-LUX-ZB 
- [#4433](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4433) Expose `indicator_mode` for TuYa TS0121_plug
- [#1475](https://github.com/Koenkk/zigbee2mqtt.io/pull/14750) Support get for Ubisys `configure_device_setup`
- [#4442](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4442) Support fan mode for HKGK BAC-002-ALZB
- [#4449](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4449) Prefer HS color mode for more accurate color for OSRAM 4058075047853
- [#4446](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4446) Expose `running_state` and increase temperature range for Danfoss Icon
- [#4450](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4450) Add OTA support for Xiaomi `ZNCZ12LM`
- [#4464](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4464) Support cooling for SMaBiT AV2010/32
- [#4499](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4499) Expose `auto_relock_time` and `sound_volume` and support OTA for Home Control AS HC-SLM-1

# Fixes
- #12710 Detect `_TZ3000_kmh5qpmb` as NAS-PD07 instead of NAS-AB02B2
- #13026 Fix TuYa TS0601_temperature_humidity_sensor (`_TZE200_bjawzodf`) humidity off by 10x
- #13054 Remove unsupported `color_temp_startup` from Gledopto GL-S-004Z
- #12999 Fix crash when `ZIGBEE2MQTT_CONFIG_ADVANCED_NETWORK_KEY` environment variable is set to `GENERATE`
- #13073 Fix crash when remove rejected device fails
- #13080 Fix Home Assistant discovery error: `expected NumberDeviceClass or one of 'temperature' for dictionary value @ data['device_class']`
- #12543 Fix no energy measurements for Hive 1613V and remove unsupported temperature
- #13028 Fix group and device states changing incorrectly when cached states are published
- #13126 Fix calibration not working for power/current/voltage for phase b and c
- #13154 Remove `color_temp_support` for all Ledvance/OSRAM/Sylvania lights since devices do not seems to support this
- #12946 Fix no `on`/`off` action for Iluminize 511.344
- #13138 Fix Moes TS0112 not updating status when physical button is pushed
- #13218 Fix no state updates when physically controlling TuYa `TS011F_2_gang_wall`
- #13159 Fix no action for TS0215A_remove (`_TZ3000_tj4pwzzm`)
- #12526 Fix Ubisys OTA updates not working
- [#4471](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4471/) Fix TuYa `_TZ3000_dowj6gyi` detected as WSD500A instead of IH-K009
- #13253 Remove `device_temperature` from Xiaomi JY-GZ-01AQ
- #9629 Fix devices unavailable when Home Assistant and MQTT broker is restarted at the same time
- #13258 Fix reported power value incorrect for SMaBiT AV2010/25
- [#4481](https://github.com/Koenkk/zigbee-herdsman-converters/issues/4481) Fix Legrand 412015 losing configured reportings
- [#13302](https://github.com/Koenkk/zigbee2mqtt/discussions/13302) Expose `power` and `energy` for Sengled E11-N1EA
- [#4491](https://github.com/Koenkk/zigbee-herdsman-converters/pull/4491) Fix Ubisys OTA not working
- #13075 Fix no energy value for LELLKI WK34-EU
- #13278 Fix `moving` not separated per endpoint for Lonsonho TS130F_dual
- #13337  Fix `max_temperature` range error for Moes BRT-100-TRV

# New supported devices
This release adds support for **63** new devices:

- `03981` Vimar IoT connected relay module
- `07043M` Immax Motion sensor
- `158-01` Plugwise Lisa zone thermostat
- `35144001` Rademacher addZ white + colour
- `3RSB22BZ` Third Reality Smart button
- `4058075208391` LEDVANCE SMART+ lamp E27 RGBW
- `4256050-RZHAC` Centralite 3-Series smart outlet appliance module
- `4512708` Namron Zigbee LED dimmer
- `8718696170656` Philips Hue White Fuzo outdoor floor light
- `8719514382350` Philips Akari downlight
- `9125` LED Trading FOH smart switch
- `915005987001` Philips Hue White and color ambiance Gradient Signe table lamp (black)
- `9290024783` Philips Hue Filament White Ambiance A60/E27 Bluetooth
- `9290024796` Philips Hue Filament White E12
- `929003045201` Philips Hue White and Color Ambiance GU10 (Centura round white)
- `929003056001` Philips Hue white ambiance Adore bathroom mirror light with Bluetooth
- `929003098601` Philips Hue Calla outdoor Pedestal
- `9290031346` Philips Hue white and color ambiance 5/6" retrofit recessed downlight
- `AJ_RGBCCT_CTRL` Ajax Online Smart Zigbee LED strip RGB+CCT
- `AX_Valve_Controller` CTM Lyng AX Valve Controller, water shutoff valve controller
- `AX_Water_Sensor` CTM Lyng AX Water Sensor, water leakage detector
- `BN-600110` Dresden Elektronik Zigbee 3.0 dimm actuator
- `DD10Z` ORVIBO Smart spotlight
- `DL001` TERNCY Two color temperature intelligent downlight
- `EFEKTA_CO2_Smart_Monitor` Custom devices (DiY) [EFEKTA CO2 Smart Monitor, ws2812b indicator, can control the relay, binding](https://efektalab.com/CO2_Monitor)
- `EFEKTA_iAQ` Custom devices (DiY) [CO2 Monitor with IPS TFT Display, outdoor temperature and humidity, date and time](http://efektalab.com/iAQ)
- `FUT066Z` MiBoxer RGB+CCT LED Downlight
- `FWGU10Bulb02UK` Hive GU10 warm white
- `GL-G-003P` Gledopto 7W garden light pro
- `H1` Ubisys Heating regulator
- `HA-ZM12/24-1K` Halemeier 1-channel smart receiver
- `HG08383A` Lidl Livarno outdoor LED light chain
- `HS3AQ` HEIMAN Smart air quality monitor
- `IA-CDZFB2AA007NA-MZN-02` ADEO ENKI LEXMAN E27 LED white
- `IC-CDZFB2AC005HA-MZN` ADEO ENKI LEXMAN E14 LED white
- `IG-CDZB2AG009RA-MZN-01` ADEO ENKI LEXMAN E27 Led white bulb
- `IG-CDZFB2G009RA-MZN-02` ADEO ENKI LEXMAN E27 LED white filament 1055 lumen
- `IH-K009` TuYa Temperature & humidity sensor
- `KCTW1Z` TuYa Temperature & humidity sensor with LCD
- `LDSENK07` ADEO ENKI LEXMAN wireless smart outdoor siren
- `MBD-S` CTM Lyng MBD-S, motion detector with 16A relay
- `PSE03-V1.1.0` EVOLOGY Sound and flash siren
- `ROB_200-026-0` ROBB 2-gang in-wall switch
- `SJCGQ13LM` Xiaomi Aqara E1 water leak sensor
- `SMARTZBA60RGBW` LEDVANCE SMART+ lamp E26 RGBW
- `SPP04G` Mercator Ikuü Quad Power Point
- `SR-ZG9092A` Sunricher Touch thermostat
- `TAFFETAS2` Acova Taffetas 2 heater
- `X5H-GB-B` TuYa Wall-mount thermostat
- `ZB-CL01` KURVIA GU10 GRBWC built from AliExpress
- `TS011F_relay_switch` TuYa Dry contact relay switch
- `ZG_LED_DRIVER42CC` ENVILAR Zigbee LED driver
- `ZL00030014` Linkind Zigbee LED 4.8W GU10 bulb, dimmable & tunable
- `ZW-EC-01` Nue / 3A Zigbee smart curtain switch
- `ZW-EU-4C` Nue / 3A Zigbee smart curtain switch
- `mKomfy_Sensor` CTM Lyng mKomfy, stove guard
- `mStikk_Outlet` CTM Lyng mStikk OP, wall socket
- `mSwitch_Mic` CTM Lyng Mikrofon, alarm detection microphone
- `mTouch_Air_Sensor` CTM Lyng Air Sensor, temperature & humidity sensor
- `mTouch_Astro` CTM Lyng mTouch Astro OP, astro clock
- `mTouch_Bryter` CTM Lyng mTouch Bryter OP, 3 channel switch
- `mTouch_Dim` CTM Lyng mTouch Dim OP, touch dimmer
- `mTouch_One` CTM Lyng mTouch One OP, touch thermostat

<hr /><em>This discussion was created from the release <a href='https://github.com/Koenkk/zigbee2mqtt/releases/tag/1.27.0'>1.27.0</a>.</em>

**Comment by ghost** [comment]
I don't update even under torture ...
With each new release something stops working ...

**Reply by roumano** [reply]
Your comment is very negative and you didn't add any facts or create issue to improve the situation.
So i think you should :
Stop using zigbee2mqtt if you are not happy, other zigbee software exist

But personally, it's now more than 1year i'm making monthly update without issue's.

**Reply by MattL0** [reply]
No issue here with a sonoff 3.0 2652p. Actually more stable than deconz I used for 4-5 years.
